lib/src/rules/prefer_initializing_formals.dart

```diff
@@ -88,14 +88,14 @@ Iterable<ConstructorFieldInitializer>
             ConstructorDeclaration node) =>
         node.initializers.whereType<ConstructorFieldInitializer>();
 
-Element _getLeftElement(AssignmentExpression assignment) =>
+FieldElement _getLeftElement(AssignmentExpression assignment) =>
     DartTypeUtilities.getCanonicalElementFromIdentifier(
         assignment.leftHandSide);
 
 Iterable<Element> _getParameters(ConstructorDeclaration node) =>
     node.parameters.parameters.map((e) => e.identifier.staticElement);
 
-Element _getRightElement(AssignmentExpression assignment) =>
+VariableElement _getRightElement(AssignmentExpression assignment) =>
```

Comment:
I don't think this is a safe change to make either. The right hand side of an assignment isn't always a variable. For example, consider a tear-off, which I believe will resolve to an ExecutableElement.

Reply:
Yep added as a check before using it instead.